sys-boot/lilo/lilo-24.2-r3.ebuild

```
@@ -0,0 +1,197 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit optfeature toolchain-funcs
+
+DOLILO_V="0.6"
+
+DESCRIPTION="LInux LOader, the original Linux bootloader"
+HOMEPAGE="https://www.joonet.de/lilo/"
+
+DOLILO_TAR="dolilo-${DOLILO_V}.tar.bz2"
+SRC_URI="
+	https://www.joonet.de/lilo/ftp/sources/${P}.tar.gz
+	mirror://gentoo/${DOLILO_TAR}
+"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+
+IUSE="static minimal pxeserial device-mapper"
+
+DEPEND="sys-devel/bin86"
+RDEPEND="device-mapper? ( sys-fs/lvm2 )"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-24.2-Makefile-tweaks.patch
+	"${FILESDIR}"/${PN}-24.2-add-nvme-support.patch
+	"${FILESDIR}"/${PN}-24.x-fix-gcc-10.patch
+	"${FILESDIR}"/${PN}-24.x-check-for-__GLIBC__.patch
+	"${FILESDIR}"/${PN}-24.2-fix-gcc-15.patch
+	"${FILESDIR}"/${PN}-24.2-musl.patch
+)
+
+# Bootloaders should not be using arbitrary CFLAGS without good reason.  A bootloader
+# is typically only executed once to boot the system, and it should work the first time.
+QA_FLAGS_IGNORED="/sbin/lilo"
```

Sorry, should've noticed this before, and really, we should make pkgcheck warn on this, but QA_* paths have to be relative:
```suggestion
QA_FLAGS_IGNORED="sbin/lilo"
```